src/libs/Navigation/AppNavigator/Navigators/LeftModalNavigator.tsx

```diff
@@ -33,11 +35,11 @@ function LeftModalNavigator({navigation}: LeftModalNavigatorProps) {
                 <Stack.Navigator screenOptions={screenOptions}>
                     <Stack.Screen
                         name={SCREENS.LEFT_MODAL.CHAT_FINDER}
-                        component={ModalStackNavigators.ChatFinderModalStackNavigator}
+                        component={ChatFinderPage}
```

Comment:
I think `getComponent` here is more performance than `component` like you do below